src/skmultiflow/trees/stacked_single_target_hoeffding_tree_regressor.py

```diff
@@ -203,19 +211,35 @@ def predict(self, X):
         list
             Predicted target values.
         """
-        r, _ = get_dimensions(X)
+        if not self._n_targets_set:
+            raise RuntimeError(
```

Comment:
Same comment as before. Better return a default value instead of raising an error.

Reply:
Agreed!

Reply:
This change was covered in commit [384ef1b](https://github.com/scikit-multiflow/scikit-multiflow/pull/168/commits/384ef1bac5731aaf46726d78997f351f85a1aac4). Thank you @jacobmontiel for your suggestion.